gateways/authorizenet/request_builders.go

```diff
@@ -43,15 +42,13 @@ func buildAuthRequest(merchantName string, transactionKey string, authRequest *s
 	var transactionRequest TransactionRequest
 	if authRequest.Options[sleet.GooglePayTokenOption] != nil {
 		// Google Pay request
-		googlePayToken := authRequest.Options[sleet.GooglePayTokenOption].(string)
-		encodedGooglePayToken := base64.StdEncoding.EncodeToString([]byte(googlePayToken))
 		transactionRequest = TransactionRequest{
 			TransactionType: TransactionTypeAuthOnly,
 			Amount:          &amountStr,
 			Payment: &Payment{
 				OpaqueData: &OpaqueData{
 					DataDescriptor: GooglePayPaymentDescriptor,
-					DataValue:      encodedGooglePayToken,
+					DataValue:      authRequest.Options[sleet.GooglePayTokenOption].(string),
```

Comment:
I think you might also need to update your unit test?

Reply:
Forgot to push commit oops